src/doc_builder/convert_md_to_mdx.py

```diff
@@ -101,6 +101,41 @@ def clean_doctest_syntax(text):
     return text
 
 
+_re_literalinclude = re.compile(r"([ \t]*)<literalinclude>(((?!<literalinclude>).)*)<\/literalinclude>", re.DOTALL)
+
+
+def convert_literalinclude_helper(match, page_info):
+    """
+    Convert a literalinclude regex match into markdown code blocks by opening a file and
+    copying specificed start-end section into markdown code block.
+    """
+    literalinclude_info = eval(match[2].strip())
```

Comment:
This will evaluate any code any user has put between litteral include tags on any of their PRs, so not super safe. Could we use something like `json.loads` that is safer?

Reply:
indeed :) working on it

Reply:
handled in https://github.com/huggingface/doc-builder/pull/179/commits/7a64237749cbbd608a5f0523b8456ead97718429